docs/cni/kubernetes/AWS.md

```diff
@@ -176,7 +176,12 @@ aws ec2 run-instances \
 #  Include the subnet param above if using a non-default VPC
```

Comment:
Can you add names to instances in this command?  I'm having difficulty telling my workers and master apart.